include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -142,40 +142,43 @@ using __kernel_name_provider =
     __optional_kernel_name<_CustomName>;
 #endif
 
-template <char...>
-struct __composite_kernel_name
+template <typename _KernelName, int...>
```

Comment:
The idea was to just drop `(char)78, ...` cast from the kernels that tools (VTune) show and leave just a number. It works in the compiler-generated string but unfortunately doesn't in VTune. There is nothing terrible if we leave `int` instead of `char`. However, I agree that we shouldn't make changes that don't bring significant value. Thanks for the reminder.

Reply:
done.